### PR TITLE
feat: Adding the operator settings for operator in dev environment

### PR DESCRIPTION
**Pull request by NitishAroraKPMG**
## Description

Creating the operator settings for the development environment

## Testing instructions

Create the local environment and run it using `make run`

### CODE REVIEW

File: fdbt-dev/sql/operatorGroup.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+LOCK TABLES `operatorGroup` WRITE;
+
+TRUNCATE TABLE `operatorGroup`;
+
+INSERT INTO `operatorGroup` (name, contents, nocCode) 
+VALUES
+('test', '[{"nocCode":"LNUD","name":"The Blackburn Bus Company"},{"nocCode":"NWBT","name": "Pilkingtonbus"}]', 'BLAC');
```
Comment (alainajaffer) [inline fdbt-dev/sql/operatorGroup.sql:8]: do we want to add this with the name test if the tests add one with this name?

File: fdbt-dev/sql/passengerTypes.sql [hunk 2]
```diff
@@ -0,0 +1,14 @@
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+LOCK TABLES `passengerType` WRITE;
+
+TRUNCATE TABLE `passengerType`;
+
+INSERT INTO `passengerType` (name,contents,isGroup,nocCode) 
+VALUES
+('Small People', '{"id":"","passengerType":"child","ageRangeMin":"","ageRangeMax":"18"}', 0, 'BLAC'),
```
Comment (RobCattonKPMG) [inline fdbt-dev/sql/passengerTypes.sql:8]: Why are we leaving ID from the Insert and from the contents?

File: fdbt-dev/sql/operatorGroup.sql (hunk 1 above)
Comment (NitishAroraKPMG) [reply fdbt-dev/sql/operatorGroup.sql:8]: let me change to operator-group

File: fdbt-dev/sql/passengerTypes.sql (hunk 2 above)
Comment (NitishAroraKPMG) [reply fdbt-dev/sql/passengerTypes.sql:8]: id we can't insert as this is auto increment and the contents contain the data in this format ...

Comment (NitishAroraKPMG) [reply fdbt-dev/sql/passengerTypes.sql:8]: i created the passenger type from the UI and use that one ... I have to check if we delete the id from single passenger type then it should not break

**Review by RobCattonKPMG — APPROVED** [review]

**Pull request merged**